frontend/src/app/features/hal/resources/time-entry-resource.ts

```diff
@@ -39,7 +40,7 @@ export class TimeEntryResource extends HalResource {
 
   comment:Formattable;
 
-  workPackage:WorkPackageResource;
+  entity:WorkPackageResource|MeetingResource;
```

Comment:
Can't we just specify the general type?

```suggestion
  entity:HalResource;
```

Reply:
Why should we do that? There is a limited set of resources it can be, why should we make it more broadly in the JS world?